### ISSUE TITLE
Delayed Start - Wait x minutes after bed temp reached before starting to print 

### ISSUE THREAD

**Issue by parrothd** [bambulab/BambuStudio#2844]
Printing in ABS or PETG often have to turn the bed on and wait a 10+ minutes so the chamber is nice and warm. 

It would be nice: 

If you could hit print and have the option to heat bed X minutes, then start printing.

Also, an option to starting printing at a time so you could hit start printing but know it will kick off at 1am.

**Comment by RalphNoordanus** [comment]
I agree, this would be a nice feature. Love to see that you can add a pause to the G-code with a simple button click.
